refactor:`hpmn` rename to `evo` (#10)
dashpay/FAQs: Platform/platform-activation-hpmn-faq.md

No, one of the tradeoffs for supporting efficient proofs using threshold cryptography is that all validators must be of the same weight in the system. As such, this breaks the delegated proof of stake model. Since all Masternodes running Platform would hold the same collateral this is not a problem.

# EvoNode Questions

## General

> 25. **What is a High-Performance Masternode (EvoNode)?**

Reviewer: ```suggestion
25. **What is an Evolution Node (EvoNode)?**
```

An Evolution Node is a proposed new type of Masternode which would be used to serve the network by participating in consensus on both the Dash Platform chain and the Dash Payment (Core) chain. In the solutions that use EvoNodes the standard Masternode would continue serving only the Dash Payment chain. EvoNodes would have greater requirements than a standard Masternode regarding collateral amount and would require higher performance specs as they would be running two chains instead of just one.

> 26. **What are the main advantages of choosing one of the High-Performance Masternode solutions?**

Reviewer: ```suggestion
26. **What are the main advantages of choosing one of the Evolution Node solutions?**
```

33. **Will EvoNodes generate revenue from Platform fees?**

Yes, fees will generate revenue except in one extreme case. If the price of Dash were to go below half the average price that was paid in Dash per byte for all bytes stored in the Platform state, revenue generation would be lower than storage hosting costs. Adjustments to the fee multiplier (originally set to 2) could cause this to fluctuate to more than or less than the "half" from the previous statement.

Satoshi’s vision for Bitcoin was that the source of system revenue would transition from block rewards to network fees. The same applies to Dash. Dash has the opportunity to become a business where users and enterprises pay for the service and the Dash network actually makes money on every transaction.

34. **In the EvoNode solutions would standard Masternodes generate revenue from Platform fees?**

They are expected to generate revenue indirectly from Platform because the equilibrium described in question 24 that exists between EvoNode rewards and normal MN rewards would be driven higher as more Platform fees are generated.

## Technical

35. **What are the targeted specs for EvoNodes?**

> | | Every node must run Platform | High Performance 4k | High Performance 10k

Reviewer: ```suggestion
| | Every node must run Platform | Evolution Node 4k | Evolution Node 10k
```
